parsec/backend/asgi/administration.py

```diff
@@ -149,3 +152,64 @@ async def administration_organization_stat(raw_organization_id: str):
         },
         status=200,
     )
+
+
+@administration_bp.route("/administration/stats", methods=["GET"])
+@administration_authenticated
+async def administration_server_stats():
+    backend: "BackendApp" = g.backend
+
+    for arg in ["format", "from"]:
```

Comment:
`from` argument as a default value from the issue

>  very beginning of the server for `--from`

For me the default value should be handled in the API not the CLI

Reply:
I missed that part of the issue. 
> not providing --from/--to with dump the stats for the entire life of the server up to now)

`from` should be defaulted as the very first day of the server and `to` to now

Reply:
yes, you already do the part for `to`